### DISCUSSION TITLE
Questions Regarding with Regard to Closing a Forked Repository; Questions Regarding Accessing Git Bash

### DISCUSSION THREAD

**Discussion by JackZ295** [community/community#31819] [Other Feature Feedback, Questions, & Ideas]
Hi! I have been looking up how to clone a forked repository on GitHub and have been following this guide: https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository. However, I am a little bit confused about Step 4: Open Git Bash. How do you access Git Bash, and where can you find this feature on Git Hub? I tried looking it up but not find anything about it. Any input regarding this would be much appreciated! Thanks so much!

**Comment by airtower-luna** [comment]
> How do you access Git Bash, and where can you find this feature on Git Hub?

It's not on GitHub, it might be on your computer. What you need there is a terminal to enter commands (like `git clone`). There's a variety of terminals on different operating systems, Git Bash is a version of the Bash shell that usually gets installed along with Git for Windows.

So: If you use Windows and have Git installed, Git Bash should be available on your system.
